src/EA.Weee.Web.Tests.Unit/Areas/Admin/Mapping/AatfDataToAatfDetailsViewModelMapTests_2.cs

```diff
@@ -0,0 +1,60 @@
+﻿namespace EA.Weee.Web.Tests.Unit.Areas.Admin.Mapping
```

Comment:
Got 2 view model tests, reminder to remove 1 of them

Reply:
Removed